core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -553,8 +553,9 @@ public void start(Boolean isPrimary) throws IOException {
         if (SecurityUtils.isSecurityEnabled() && !root.getOwner().isEmpty()
             && !root.getOwner().equals(serverOwner)) {
           // user is not the previous owner
-          throw new PermissionDeniedException(
-              ExceptionMessage.PERMISSION_DENIED.getMessage("Unauthorized user on root"));
+          throw new PermissionDeniedException(ExceptionMessage.PERMISSION_DENIED.getMessage(String
```

Comment:
this is an odd mix of `ExceptionMessage` with hardcoding the message. I'd prefer either putting the full error string here, or moving all the text to `ExceptionMessage`